sdk/helper/template/template.go

```diff
@@ -73,6 +84,10 @@ func Function(name string, f interface{}) Opt {
 //   - base64 encodes the previous value.
 //     Example: {{ .DisplayName | base64 }}
 //
+// - unbase64
```

Comment:
nitpic: something like `base64dec` would be a more familiar naming

Reply:
Yeah, the existing one isn't `base64enc` tho, so I feel weird naming it `base64dec`... Maybe `decbase64`?

Reply:
Another function is called `truncate_sha256, so how about `decode_base64` ?

Reply:
Nice, I like that one!